cmd/prometheus/main.go

```diff
@@ -417,7 +417,7 @@ func main() {
 	serverOnlyFlag(a, "rules.alert.resend-delay", "Minimum amount of time to wait before resending an alert to Alertmanager.").
 		Default("1m").SetValue(&cfg.resendDelay)
 
-	serverOnlyFlag(a, "rules.max-concurrent-evals", "Global concurrency limit for independent rules that can run concurrently.").
+	serverOnlyFlag(a, "rules.max-concurrent-evals", "Global concurrency limit for independent rules that can run concurrently.When changing the value for rules.max-concurrent-evals please adjust the value for query.max-concurrency as the rules manager uses the same query engine than the API:").
```

Comment:
```suggestion
	serverOnlyFlag(a, "rules.max-concurrent-evals", "Global concurrency limit for independent rules that can run concurrently. When set, \"query.max-concurrency\" may need to be adjusted accordingly.").
```

A small hint will be sufficient I think.

Reply:
yeah you are right @machine424